EED_Espresso_Calendar.module.php

```diff
@@ -869,7 +880,9 @@ public function get_calendar_events()
 
                 $calendar_datetime = apply_filters('FHEE__EE_Calendar__get_calendar_events__calendar_datetime', $calendar_datetime, $datetime);
 
-                $calendar_datetimes_for_json [] = $calendar_datetime->to_array_for_json();
+                if ( $calendar_datetime ) { 
```

Comment:
great addition 💯 

should never trust results from a filter.

that said, an instanceof check is better because it proves the existence of the exact object we want

```suggestion
                if ($calendar_datetime instanceof EE_Datetime_In_Calendar) { 
```